### ISSUE TITLE
API to retrieve the fingerprint information

### ISSUE THREAD

**Issue by bastiao** [bioinformatics-ua/catalogue#1011]
- This API should be well documented.
- By default it should be only be available for the database owner. 
- groups of admins (certain group) could access for more services. 

We may need to create the notion of API Key and Secret Key.

**Comment by ribeiroct** [comment]
Well we already have it in the Singletons, which can be consumed by plugins and are well documented. Maybe opening them their API up through the public keys also ?

Their api's are providing the required information, yes ?

P.e.:
developer/api/getFingerprintUID/<fingerprint_hash>
developer/api/getAnswers/<fingerprint_hash>

**Comment by bastiao** [comment]
Yes, this exactly what we want. We need to allow for a API Key that can consume the data (only, and only if the user in a certain group).

By the way, in Extra Fields the GET API need to be redefined.
